lib/python/qmk/tests/test_cli_commands.py

```diff
@@ -45,3 +45,8 @@ def test_list_keyboards():
     # check to see if a known keyboard is returned
     # this will fail if handwired/onekey/pytest is removed
     assert 'handwired/onekey/pytest' in result.stdout
+
+def test_list_keymaps():
+    result = check_subcommand("list_keymaps", "-kb", "planck/ez")
```

Comment:
This should be `list-keymaps`, no?

Reply:
Since your command has some error handling, I'd also recommend adding a test for that case. Something like:

`def test_list_keymaps_no_keyboard_found():
    result = check_subcommand("list-keymaps", "-kb", "asdfghjkl")
    assert result.returncode == 0
    assert "does not exist" in result.stdout`

Reply:
You are right. I haven't updated the code after #7025 got merged.
Also, thanks for the additional test case.